cve_bin_tool/checkers/sqlite.py

```diff
@@ -155,8 +185,9 @@ def get_version_map():
         print("Could not fetch " + changeurl + ", " + err)
     lines = response.readlines()
 
-    version_pattern = re.compile(r'<h3>\d{4}-\d{2}-\d{2} \((\d+\.\d+\.?\d?)\)</h3>')
-    id_pattern = re.compile(r'SQLITE_SOURCE_ID: "([^"]+)"')
+    version_pattern = re.compile(r'<h3>\d{4}-\d{2}-\d{2} \((\d+\.\d+[.\d]*)\)</h3>')
+    id_patterns = [re.compile(r'SQLITE_SOURCE_ID: "([^"]+)"'),
+                   re.compile(r'"*(\d{4}-\d{2}-\d{2} \d+:\d+:\d+ [\w]+)"*')]
```

Comment:
I have added two patterns.  The first original will catch most of the source_id's .   
In situations where 
1. `SQLITE_SOURCE_ID` and the values are in separate lines
2.  The `SQLITE_SOURCE_ID` values not starting with double quotes 
 will be caught by the second pattern.